rush.json

```diff
@@ -421,7 +421,7 @@
     },
     {
         "packageName": "@binaris/utils-subprocess",
-        "projectFolder": "tools/subprocess",
+        "projectFolder": "utils-subprocess",
```

Comment:
This is actually not recommended if following "category folders" recommendation from https://github.com/binaris/shiftjs/blob/120f401bc2cffa0be736ffb7cd047cd3d3a46d91/rush.json#L88

Reply:
Let's leave it for now.